From e842f8f710fbfc721c4ebf5c2a4cbe27feac1a05 Mon Sep 17 00:00:00 2001
From: "wingate.chi" <wingate.chi@cybertan.com.tw>
Date: Tue, 4 Feb 2025 06:32:17 +0000
Subject: [PATCH 37/55] fstools: increase rootfs size to 500M

Increases F2FS minimum filesystem size to 500M to accommodate larger firmware images and package installations.

Signed-off-by: wingate.chi <wingate.chi@cybertan.com.tw>
---
 .../patches/0001-change-rootfs-size-to-500m.patch     | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 package/system/fstools/patches/0001-change-rootfs-size-to-500m.patch

diff --git a/package/system/fstools/patches/0001-change-rootfs-size-to-500m.patch b/package/system/fstools/patches/0001-change-rootfs-size-to-500m.patch
new file mode 100644
index 0000000000..f03adc5854
--- /dev/null
+++ b/package/system/fstools/patches/0001-change-rootfs-size-to-500m.patch
@@ -0,0 +1,11 @@
+--- a/libfstools/common.h
++++ b/libfstools/common.h
+@@ -19,7 +19,7 @@
+ #include "libfstools.h"
+ #include "volume.h"
+ 
+-#define F2FS_MINSIZE		(100ULL * 1024ULL * 1024ULL)
++#define F2FS_MINSIZE		(500ULL * 1024ULL * 1024ULL)  //change rootfs size to 500M
+ 
+ int read_uint_from_file(char *dirname, char *filename, unsigned int *i);
+ char *read_string_from_file(const char *dirname, const char *filename, char *buf, size_t bufsz);
-- 
2.34.1

